vmdb/app/controllers/ems_common.rb

```diff
@@ -702,6 +702,10 @@ def get_form_vars
       @edit[:new][:emstype] = params[:server_emstype]
       if ["openstack", "openstack_infra"].include?(params[:server_emstype])
         @edit[:new][:port] = @ems.port ? @ems.port : 5000
+      elsif params[:server_emstype] == EmsKubernetes.ems_type
+        @edit[:new][:port] = @ems.port ?  @ems.port : EmsKubernetes.new.port
```

Comment:
> @abonas I think this should be a constant, EmsKubernetes::DEFAULT_PORT and have dafault_value_for use that constant. I don't see the need to instantiate an object here.
> 
> This can be fixed in a followup PR.

@jrafanie , ack. lets get this merged and I'll fix it separately. (I filed an issue so it won't be lost)